core/io/resource_loader.cpp

```diff
@@ -773,6 +783,19 @@ ResourceUID::ID ResourceLoader::get_resource_uid(const String &p_path) {
 	return ResourceUID::INVALID_ID;
 }
 
+String ResourceLoader::get_attached_script_path(const String &p_path) {
+	String local_path = _validate_local_path(p_path);
+
+	for (int i = 0; i < loader_count; i++) {
+		String path = loader[i]->get_attached_script_path(local_path);
+		if (path != "") {
```

Comment:
```suggestion
		if (!path.is_empty()) {
```
This is established way of checking if String is empty.